### PR TITLE
Fixed PSR Formatting

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/ExporterBundle/Service/DataObfuscationService.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 class DataObfuscationService
 {
     /**
-     * @param array<Questionnaire> $questionnaires
+     * @param array <Questionnaire> $questionnaires
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Service/DataObfuscationService.php]: Being picky here, but was it not originally correct with no space?

File: src/TransformCore/PHE/ExporterBundle/Service/FileCompressionService.php [hunk 2]
```diff
@@ -48,7 +48,7 @@ public function findAndCompressFiles()
      */
     protected function findFilesInPath()
     {
-        return glob($this->basePath.'*.asc');
+        return glob($this->basePath . '*.asc');
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Service/FileCompressionService.php]: Much easier to read :smile:

File: src/TransformCore/PHE/ExporterBundle/Service/DataObfuscationService.php (hunk 1 above)
Comment (brettminnie) [inline src/TransformCore/PHE/ExporterBundle/Service/DataObfuscationService.php]: Not sure, this is according to phpstorm

In C++/Java you wouldn't have a space

Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Service/DataObfuscationService.php]: Yeah,  I am thinking of Java then. Don't see this much in PHP although it should be used more :wink:

**Pull request merged**